GEOSldas_App/ldas_setup

```diff
@@ -1710,6 +1728,11 @@ def parseCmdLine():
         help='replace computing/sponsor account in batinp file',
         type=str, default='None'
         )
+    p_setup.add_argument(
+        '--nymdb',
+        help='replace BEG_DATE in expinp file ',
```

Comment:
Not sure I understand the comment.  Should this read:
'replace**s** BEG_DATE in expinp file'    ?? (without the s in "replaces", it reads like an instruction, when I think this help text describes the effect of specifying --nymdb
Also, it looks like there's a typo:  "exeinp" instead of "expinp"?

Reply:
spot on.   maybe we can say "supersedes BEG_DATE in expinp file "